exec_manager/dao/db_models.py

```diff
@@ -15,26 +15,22 @@
 
 """Defines all database specific ORM models"""
 
-from sqlalchemy import JSON, Boolean, Column, Integer, String
+
+from sqlalchemy import JSON, Column, String
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm.decl_api import DeclarativeMeta
```

Comment:
Du kannst den UUID Typ von SQLalchemy benutzen:

```suggestion
import uuid

from sqlalchemy.orm.decl_api import DeclarativeMeta
from sqlalchemy.dialects.postgresql import UUID
```